src/components/PostListVolunteer/index.css

```diff
@@ -0,0 +1,54 @@
+@import url("https://fonts.googleapis.com/css?family=Acme&display=swap");
+
+.searchAndFilter {
+  border: 3px solid;
+  font-family: Acme;
+  color: #590995;
+  border-color: #590995;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  align-content: center;
+  padding: 1%;
+}
+
+label {
```

Comment:
Can you give a className for the label so it doesn’t affect all the labels in the site